Lib/test/test_capi/test_list.py

```diff
@@ -0,0 +1,85 @@
+import unittest
+import sys
+from test.support import import_helper
+from collections import UserList
+
+_testcapi = import_helper.import_module('_testcapi')
+
+NULL = None
+
+class ListSubclass(list):
+    ...
+
+
+class CAPITest(unittest.TestCase):
+    def test_check(self):
+        # Test PyList_Check()
```

Comment:
Please add similar comment in every method. It helps to search what functions were tested.

Reply:
Thank you. I will make sure to add the C API in the comment